src/client/containers/dashboard-page/dashboard-page.style.css

```diff
@@ -0,0 +1,62 @@
+.dashboard {
+  display: flex;
+  justify-content: flex-start;
+  min-height: 100vh;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  justify-content: flex-start;
+  margin-left: 130px;
+  width: 100%;
+}
+
+header {
+  padding: 1em 1em 0;
+}
+
+header h1 {
+  font-size: 1.3em;
+  margin: 1em 0 0;
+  padding: 0 1em;
+  text-transform: uppercase;
+}
+
+.dashboard main {
+  align-items: flex-start;
+  display: flex;
+  flex-direction: column;
+  height: auto;
+  justify-content: flex-start;
+  padding: 0 1em;
+  width: 100%;
+}
+
+.dashboard .cards {
+  display: flex;
+  flex-flow: wrap;
+  justify-content: space-between;
+  margin: 2em 0;
+}
+
+.cards .card {
+  margin: 1em 1.2em;
+  min-width: 19.5em;
+}
+
+.cards .card:first-child {
+  background: rgba(255, 255, 255, 0.5);
+  box-shadow: none;
+}
+
+.cards .card:first-child h3 {
+  margin-bottom: 0;
+}
+
+.dashboard .modal > .content {
+  display: block;
+  height: auto;
+  margin-left: 0;
+}
```

Comment:
This is quite well written and is mainly concerned with organizing the existing components. Good work 💪 

Note: This needs to live in the stateless `<DashboardPage>` component, not the container. So the stateless components that come with all the markup also have all of the associated styling that goes with the markup.